pkg/cmd/pipelinerun/list_test.go

```diff
@@ -501,3 +537,16 @@ func command(t *testing.T, prs []*v1alpha1.PipelineRun, now time.Time, ns []*cor
 
 	return Command(p)
 }
+
+func commandV1beta1(t *testing.T, prs []*v1beta1.PipelineRun, now time.Time, ns []*corev1.Namespace, version string, dc dynamic.Interface) *cobra.Command {
+	// fake clock advanced by 1 hour
+	clock := clockwork.NewFakeClockAt(now)
+	clock.Advance(time.Duration(60) * time.Minute)
+
+	cs, _ := test.SeedV1beta1TestData(t, pipelinev1beta1test.Data{PipelineRuns: prs, Namespaces: ns})
+	cs.Pipeline.Resources = cb.APIResourceList(version, []string{"pipelinerun"})
+
+	p := &test.Params{Tekton: cs.Pipeline, Clock: clock, Kube: cs.Kube, Dynamic: dc}
+
+	return Command(p)
+}
```

Comment:
Any reason this couldn't be a single function that just accepts the version as a param as far as what to return for the command?

Reply:
I think, we can go with the approach you suggested. Thanks for the suggestion :)

Reply:
@danielhelfand as I have gone through the code, I have noticed that there is no `seedTestData` func in `v1beta1` package. So can we go with this approach i.e. having two different func `commandV1alpha1` and `commandV1beta1`?
Please let me know your thoughts on this.

Reply:
Right, you would use `test.SeedV1beta1TestData`. In reflecting back on this, I see why two funcs makes sense though, so feel free to leave as is. Thanks for trying it though.

Reply:
Okay :slightly_smiling_face: